zlib-rs/src/deflate.rs

```diff
@@ -1226,7 +1216,7 @@ impl<'a> BitWriter<'a> {
     }
 }
 
-#[repr(C)]
+#[repr(C, align(64))]
```

Comment:
it's fine to have this, because that is the intention, but the state is always stored on the heap, and we make sure that all allocations are aligned to 64 bytes anyway, so in practice this does not have any effect.